runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -254,7 +254,6 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 		return getSpineSizeWithoutHeader(layout, numberArraylets, dataSize, alignData);
 	}
 
-#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
```

Comment:
Comment below should be edited to eliminate double map mentioning. We can do it later, but if we remove ifdef now we can miss it at the final double map removing step.